bond_data.py

```diff
@@ -24,11 +24,12 @@ def field_mapping(df, maps):
     Returns: df with renamed columns
     -------
     """
-    cols = df.columns
 
     # Making sure that all columns are present
```

Comment:
maybe the comment could be "making sure the necessary columns are present"

but it still makes sense either way